src/main.rs

```diff
@@ -139,7 +139,10 @@ fn deploy_parser(
         Address(0.into()),
     )?;
 
-    cached_state.deploy_contract(Address(address.clone()), string_to_hash(&args.class_hash))?;
+    cached_state.deploy_contract(
+        Address(address.clone()),
+        string_to_hash(&args.class_hash).unwrap(),
```

Comment:
why unwrap?

Reply:
Because it expects an array `[u8; 32]` and without the unwrap it gets an enum `Result<[u8; 32], ParseFeltError>`.
Should I go about this in a different way?

Reply:
Done!